tests/test_docker_retry.py

```
@@ -0,0 +1,222 @@
+
+@pytest.fixture
+def mock_docker_client():
+    """Create a mock Docker client."""
+    with patch("mcpbr.docker_env.docker.from_env") as mock_from_env:
+        mock_client = MagicMock()
+        mock_from_env.return_value = mock_client
+        yield mock_client
+
+
+@pytest.fixture
+def manager(mock_docker_client):
```

Ruff (0.14.14)

[warning] Unused function argument: `mock_docker_client`

(ARG001)